index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <h1 id="title">Dr. Norman Bourlag</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+          alt="Dr. Normat talking"
+        />
```

Comment:
Kindly consider wrapping these tags with a header tag.

Reply:
👏 Great job on providing feedback! I appreciate your efforts in giving suggestions to improve the code. To further enhance the quality of your comments, I would suggest the following:

✨ Maintain a polite and constructive tone: Instead of using imperative statements, try phrasing your comments as suggestions to maintain a respectful and collaborative atmosphere.

🔍 Provide explanations for your requests: It's important to include the 'why' behind your suggested changes. This helps the developer understand the reasoning and importance behind the requested modification.

💡 Ensure clarity in your comments: Make sure your comments are clear and easy to understand. Provide specific details and examples to help the developer grasp your suggestions more effectively.

By following these guidelines, you'll be able to provide feedback that is both helpful and aligned with the established communication rules. Keep up the great work! 💪

Reply:
- [ ] Great job in sounding polite in your review. 🤗 However, as the previous reviewer mentioned above, when you give some suggestions it is advisable you provide the reason behind the request because it would help the concerned person to learn the lesson 😁

Reply:
> - [ ] Hello @JhonatanErosa, I wanted to bring to your attention that the changes requested by the previous reviewer have not been implemented yet. I kindly request you address those changes as soon as possible.👍When you explain the reasons behind a requested change, it helps the student understand the specific problem or improvement that needs to be addressed. By providing context and reasoning, you ensure that the student comprehends the purpose of the request.✔